src/lando/api/views.py

```diff
@@ -157,20 +166,99 @@ class hg2gitCommitMapView(CommitMapBaseView):
     scm = SCM_TYPE_HG
 
 
+# TODO: move all these to `lando.api.views.pull_requests`.
 class PullRequestAPIView(APIView):
     """Handle pull requests in the API."""
 
     def get(self, request: WSGIRequest, repo_name: str, number: int) -> JsonResponse:
         """Return a serialized JSON representation of a pull request."""
         target_repo = Repo.objects.get(name=repo_name)
         client = GitHubAPIClient(target_repo)
-        pull_request = PullRequest(client.get_pull_request(number))
+        pull_request = PullRequest(client.get_pull_request(number), target_repo)
+        patch = client.get_patch(number)
+        diff = client.get_diff(number)
+        return JsonResponse({"diff": diff, "patch": patch}, status=200)
         return JsonResponse(pull_request.serialize(), status=200)
 
 
-class LandingJobAPIView(View):
-    """Handle landing jobs in the API."""
+class LandingJobPullRequestAPIView(View):
+    """Handle pull request landing jobs in the API."""
+
+    def get(
+        self, request: WSGIRequest, repo_name: int, pull_number: int
+    ) -> JsonResponse:
+        """Return list of landing job ids by status."""
+
+        target_repo = Repo.objects.get(name=repo_name)
+        client = GitHubAPIClient(target_repo)
+        pull_request = PullRequest(client.get_pull_request(pull_number), target_repo)
+        landing_jobs = defaultdict(list)
+        for landing_job in pull_request.landing_jobs:
+            landing_jobs[landing_job.status].append(landing_job.id)
+
+        if len(landing_jobs[JobStatus.LANDED]):
```

Comment:
nit: I think we can just use 
```suggestion
        if landing_jobs[JobStatus.LANDED]:
```

as empty lists are `False`y